packages/medusa/src/loaders/admin.ts

```diff
@@ -1,22 +1,43 @@
 import { AdminOptions, ConfigModule } from "@medusajs/types"
 import { Express } from "express"
+import fs from "fs"
+import path from "path"
 
 type Options = {
   app: Express
-  adminConfig: ConfigModule["admin"]
+  configModule: ConfigModule
```

Comment:
```suggestion
  configModule: Pick<ConfigModule>
```

The idea being that you are not require to pass the entire configuration as only the admin is consumed

Reply:
When we have a "plugins" concept again, we will need to read that to resolve where to look for possible admin extensions. So right now we only need the admin part later we will need more :)

Reply:
oh yes thats true 😂 my bad :musc